contrib/seeds/makeseeds.py

```diff
@@ -39,16 +39,16 @@
     r"0.16.(0|1|2|3|99)|"
     r"0.17.(0|0.1|1|2|99)|"
     r"0.18.(0|1|99)|"
-    r"0.19.(0|1|2|99)|"
+    r"0.19.(0|1|2|99).(0|1)|"
```

Comment:
The regexp doesn't match the entire string until the end, so "0.19.0.1" and such already match. This change will only make it that "0.19.1" no longer matches.

Reply:
Oops -- done, thank you!